src/match.cpp

```diff
@@ -220,14 +224,12 @@ std::vector<MatchResult> regexbench::match(Engine& engine,
                                   (pLogger ? pLogger.get() : nullptr)));
   }
 
-  uint32_t sec = 0;
-  bool realTime = true;
+  gettimeofday(&begin, NULL);
```

Comment:
If begintime add in MatchResult, must be compared to begintime of all thread.
Also begintime must be updated in statistic function.
So I used local variable begin simply.

Is it better to use begintime  of all thread?